src/xc_integrator/local_work_driver/device/cuda/kernels/uvvars.cu

```diff
@@ -10,10 +10,20 @@
 #include "device_specific/cuda_device_constants.hpp"
 #include <gauxc/util/div_ceil.hpp>
 #include "device_specific/cuda_util.hpp"
+#include "device/xc_device_data.hpp"
 
 namespace GauXC {
 
-__global__ void eval_uvars_lda_kernel( size_t        ntasks,
+#define VVAR_KERNEL_SM_BLOCK 32
+#define GGA_KERNEL_SM_WARPS 16
+#define MGGA_KERNEL_SM_BLOCK 32
+
+__global__ void eval_uvars_lda_rks_kernel( size_t ntasks, XCDeviceTask* tasks_device) {
```

Comment:
I am assuming nvcc will elide this kernel. If we don't like this, it only comes at the cost of some code duplication down below.